package.json

```diff
@@ -86,6 +86,7 @@
     "d3-interpolate": "^3.0.1",
     "d3-sankey": "^0.12.3",
     "d3-scale": "^4.0.2",
+    "d3-selection": "^3.0.0",
```

Comment:
Curious why this was needed - can you provide some extra context - seems overkill for this.

Reply:
This is for using the [pointer](https://d3js.org/d3-selection/events#pointer) method so that I could grab the cursor position relative to the graph when hovering over a segment (on [this line](https://github.com/reaviz/reaviz/pull/146/files#diff-f9d90e1ec44ac4b801b822588588afc43c6218dff557478f18b16294b225a6a1R215))

I couldn't use the `event.clientX` position because it's the mouse position across the whole viewport, but there might be a way to do this without the `d3-selection` package

Reply:
Yes - I believe if you look at the zoom/pan one you can see how it uses position + scale invert to find the points.